crawl-ref/source/timed-effects.cc

```diff
@@ -1449,10 +1449,7 @@ monster* update_monster(monster& mon, int turns)
     mon.foe_memory = max(mon.foe_memory - turns, 0);
 
     if (turns >= 10 && mon.alive())
```

Comment:
I know you didn't add this code, but this could use a TODO about converting from the literal `10` to whatever constant we use for converting from turns to auts.